tests/bsim/bluetooth/audio/test_scripts/audio_config_tests/cap_ac_10_16_2_1.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "$0")/_ac_common.sh
+
+Execute_cap_unicast_ac -c 10 -r 16_2_1 -n 1
```

Comment:
Even though this is contradictory with the aim of this PR, If you want to save a few files you can do some minor groups (of say no more than 5 tests).

Reply:
I prefer the either-or approach. Making arbitrary groups based on no data seems counterintuitive to me. 

What would the advantage of creating _some_ groups be?

Reply:
Up to you. Just saving a few tenths of files :), if there was some logical way to subgroup them.